tasks/get_current_archive_list/get_current_archive_list.py

```diff
@@ -426,18 +438,23 @@ def translate_s3_import_to_partitioned_data_sql() -> text:  # pragma: no cover
     """
     return text(
         f"""
-        INSERT INTO orca.reconcile_s3_object (job_id, orca_archive_location, key_path, etag, last_update, size_in_bytes, storage_class, delete_marker)
-            SELECT 
-            :job_id, 
-            orca_archive_location, 
-            key_path, 
-            CONCAT('"', etag, '"') as etag, /* copy_to_glacier's AWS call presently wraps this in quotes. Seems like a bug, but is shown on https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html#S3.Client.list_object_versions */
-            last_update, 
-            COALESCE(size_in_bytes, 0), 
+        INSERT INTO orca.reconcile_s3_object (job_id, orca_archive_location, key_path,
```

Comment:
May want to adjust formatting here for better readability. Maybe something like
```
INSERT INTO orca.reconcile_s3_object (
    job_id,
    orca_archive_location,
    ....
)
SELECT
...
```

Reply:
updated